expertise/create_dataset.py

```diff
@@ -26,12 +26,71 @@ def convert_to_list(config_invitations):
     assert isinstance(invitations, list), 'Input should be a str or a list'
     return invitations
 
-def get_publications(openreview_client, author_id):
+def get_publications(openreview_client, config, author_id):
     content = {
         'authorids': author_id
     }
     publications = openreview.tools.iterget_notes(openreview_client, content=content)
-    return [publication for publication in publications]
+
+    # Get all publications and assign tcdate to cdate in case cdate is None. If tcdate is also None
+    # assign cdate := 0
+    unsorted_publications = []
+    for publication in publications:
+        if getattr(publication, 'cdate') is None:
+            publication.cdate = getattr(publication, 'tcdate', 0)
+        unsorted_publications.append(publication)
+
+    # If the author does not have publications, then return early
+    if len(unsorted_publications) == 0:
+        return unsorted_publications
+
+    dataset_params = config.get('dataset', {})
+    minimum_pub_date = dataset_params.get('minimum_pub_date') or dataset_params.get('or', {}).get('minimum_pub_date', 0)
```

Comment:
@carlosmondra I am not sure about this second condition in this statement. 
Is there a situation where in dataset_params we would not have 'minimum_pub_date' but a dict 'or' with the key 'minimum_pub_date' would be there?

Reply:
Right, I forgot to add to the README. Basically I am adding two configuration options:
- `minimum_pub_date`
- `top_recent_pubs`

Before you read the explanation, it might also help see the test case that I created for this function :D.

`minimum_pub_date` refers to the minimum publication date. Any submission that is older that this date will not be taken into account for the expertise.
`top_recent_pubs` can be a number or a percentage. If the user sets a number, like say 3, then only the 3 most recent publications are taken into consideration. The same applies for percentage. If the reviewer has less than 3 publications, then all his/her publications are taken into consideration. For percentages, even if we select 10%, if the user has 3 publications, then 10% of 3 is 0.3. Decimal values are always rounded to the next integer, so the result in this case would be 1.

Now, if both options are present, I didn't know if it was better to do an AND or an OR between the results of both constraints, so I decided to add both options. If these parameters are inside the `or` field, then I do an OR relationship if they are not, then I do an AND relationship.